### PR TITLE
Update the `TessQualityFlags` class with new quality flags from the mission

### PR DESCRIPTION
**Pull request by christinahedges**
The TESS SPOC have updated their quality flags, adding bits 13-15. This PR adds those new keywords along with their descriptions.

### CODE REVIEW

**Comment by christinahedges** [comment]
@barentsen I requested your review here to check there's nothing missing from this PR, thank you for any help!

**Comment by orionlee** [comment]
Close  #1163   

Comments: 

1. `HARDEST_BITMASK` should be updated accordingly.

https://github.com/lightkurve/lightkurve/blob/155708d5d52324eb190c72412c943d960f940a51/src/lightkurve/utils.py#L258-L259

2. #1163 also has some discussion on whether the default bitmask should be updated. There was no definite conclusion. It shouldn't block this PR anyway.
3.

**Comment by christinahedges** [comment]
Thanks @orionlee, maybe the TESS GI office need to have a think about what we would make the recommended bitmask for most users and check that aligns with the default. I'll update the `HARDEST_BITMASK` and then I think we can get this merged.

**Comment by orionlee** [comment]
@barentsen I think this can be merged too.

The issue about whether default bitmask needs to be changed is not a showstopper. From what I could tell, the new bits don't affect the result of PSCSAP_FLUX much anyway.

File: src/lightkurve/utils.py
Comment (barentsen) [inline src/lightkurve/utils.py:258]: Sanity check: this value is correct because ``(32768 * 2) - 1 == 65535``.

**Comment by barentsen** [comment]
> The issue about whether default bitmask needs to be changed is not a showstopper.

Agreed.  The concept of a default bitmask is getting tricky now that we have multiple pipelines which include quality flags that are often similar but not identical.  Default bitmask recommendations should probably be documented in the pipeline-specific `read_{pipeline}_lightcurve()` functions located in `io/{pipeline}.py`.  An unfortunate drawback is that these pipeline-specific reader functions are rather hidden in the call stack when opening light curves via `SearchResult.download()`.

**Review by barentsen — APPROVED** [review]
LGTM!

This fix will be part of the upcoming v2.3.0 release. I will add an entry to the change log after merging.

**Pull request merged**
